### ISSUE TITLE
Steam Deck Trackpad Doesnt produce any haptics

### ISSUE THREAD

**Issue by hectormaximillian** [Valkirie/HandheldCompanion#590]
Hello, I was using steam deck trackpad to control mouse and browse windows, but the haptics are just silent, when i close handheld companion it working again. Is theres any setting i missed to fix this?

**Comment by MetaDream-SteamDeck** [comment]
Well, have the same problem, try to reinstall HC

**Comment by CasperH2O** [comment]
@Valkirie can I get a yay or nay if this issue is still valid, I recall there was a time when this was so.

**Comment by hectormaximillian** [comment]
> @Valkirie can I get a yay or nay if this issue is still valid, I recall there was a time when this was so.

yes. Its still an issue. I never got it to work. Installed windows many times already. And just now i installed fresh windows and newest HC. Done, restarted, opened HC, enable Desktop Mode. Aaand the trackpad haptics is silent when i move the mouse

**Comment by MrSco** [comment]
Is there a fix for this?

**Comment by lyndon025** [comment]
I also have this problem. just installed handheld companion

**Comment by hectormaximillian** [comment]
> Is there a fix for this?

no, its been 2 years. I believe they dont give af. Its so annoying. Worst "companion" ever.

**Comment by CasperH2O** [comment]
> > Is there a fix for this?
> 
> no, its been 2 years. I believe they dont give af. Its so annoying. Worst "companion" ever.

@hectormaximillian Perhaps instead of being rude af to people who work on something for free in their spare time you could help out and try a more constructive way to contribute?

**Comment by MrSco** [comment]
very strange that it works without handheld companion... you'd think they should be able to just tap into that or let it pass through... not like they have to figure out the haptics, they work already with the default windows controller drivers

**Comment by Gavy310** [comment]
I don't know why Casper so butt hurt, its been two years bro.... Being frustrated that the app broke something that worked before is justified. 

Anyways brothers, simply close the steam controller program in your hidden icon menu and you haptics will work as before. 

When your ready to game, simply run the exe again to gain controller support.

That's the best I can do for you boys. Godspeed
